app/src/main/java/io/appium/settings/RecorderConstant.java

```diff
@@ -0,0 +1,14 @@
+package io.appium.settings;
+
+public class RecorderConstant {
```

Comment:
this could be an interface

Reply:
"constants in class vs constants in interface" is a lot debated topic for a long time AFAIK (https://stackoverflow.com/a/1373019) it seems interface approach is oldschool but i can do it however if that is coding convention in appium 👍